src/com/owncloud/android/providers/FileContentProvider.java

```diff
@@ -154,6 +155,57 @@
                 ProviderTableMeta.OCSHARES_ACCOUNT_OWNER);
     }
 
+    // Projection of Capabilities table
```

Comment:
I think the point of @AndyScherzinger is, this is not needed, nor for this table, nor for the others. I made a fast test commenting out the calls to ```.setProjectionMap``` that pass these maps as a parameter, and the queries work perfectly without it. As he said, we are just mapping the names to themselves again, so it results in no change.

Let's take this PR as a chance to fully remove ```mFileProjectionMap```, ```mOCSharesProjectionMap``` and ```mCapabilitiesProjectionMap```. Don't worry about QA, if this could break anything, will be easy to see in the regression tests.